src/options.js

```diff
@@ -346,6 +347,17 @@ export default class Options extends React.Component {
                             />
                         </Col>
                     </Row>
+                    <Row>
+                        <Col>
+                            <FormCheck
```

Comment:
Up to you if you want to change it, but you don't need to put strings in braces in JSX